mdk/build.gradle

```diff
@@ -143,4 +143,6 @@ publishing {
             url "file:///${project.projectDir}/mcmodsrepo"
         }
     }
-}
\ No newline at end of file
+}
+
+System.out.println("Running with java ${System.getProperty("java.version")} (${System.getProperty("os.arch")})")
```

Comment:
You should probably use a lifecycle logger here instead of the usual System.out.println

Something like project.getLogger().lifecycle(..) will do i think.

Reply:
Disregard the previous review, has been cleared up as, requested and already merged in to 1.16.